dll/win32/shell32/dialogs/filedefext.cpp

```diff
@@ -1334,6 +1408,10 @@ CFileDefExt::CountFolderAndFiles(HWND hwndDlg, LPWSTR pwszBuf, UINT cchBufMax, D
 
     do
     {
+        ZeroMemory(WorkBuffer, sizeof(WorkBuffer));
+        StringCchCopyW(WorkBuffer, sizeof(WorkBuffer), PathBuffer);
+        StringCchCatW(WorkBuffer, sizeof(WorkBuffer), wfd.cFileName);
```

Comment:
```suggestion
        StringCbCopyW(WorkBuffer, sizeof(WorkBuffer), PathBuffer);
        StringCbCatW(WorkBuffer, sizeof(WorkBuffer), wfd.cFileName);
```

Reply:
`Cch` is count of characters.
`Cb` is count of bytes.

Reply:
Yes, nice one....